src/test/java/com/google/sps/data/ReferendumTest.java

```diff
@@ -45,6 +52,28 @@ public void testAddElectionQueryData() throws Exception {
         "Water Bond. Funding for Water Quality, Supply, Treatment, and Storage Projects.");
   }
 
+  @Test
+  public void missingDescription_testFromJSONObject() throws Exception {
+    JSONObject referendumJsonObject =
+        new JSONObject("{\"type\": \"Referendum\",\"referendumTitle\": \"Proposition 1\"}");
+
+    Referendum referendum = Referendum.fromJSONObject(referendumJsonObject);
+
+    Assert.assertEquals(referendum.getTitle(), "Proposition 1");
+    Assert.assertEquals(referendum.getDescription(), "");
+  }
+
+  @Test
+  public void missingTitle_throwJSONException_testFromJSONObject() throws Exception {
+    JSONObject referendumJsonObject =
+        new JSONObject(
+            "{\"type\": \"Referendum\", \"referendumSubtitle\": \"Water Bond. Funding for Water Quality, Supply, Treatment, and Storage Projects.\"}");
+
+    exceptionRule.expect(JSONException.class);
+    exceptionRule.expectMessage("Referendum title not found in JSON response.");
```

Comment:
Prefer assertThrows:
go/java-testing/exceptions#assertThrows
https://screenshot.googleplex.com/4tnrOY4ybCJ

(lemme know if you can't access that site)

Reply:
There are a few reasons for this, but a big one is that theoretically any of the followign code could throw the exception, and this test would pass, even if it's not the actual line you intend. 

This would probably be okay in this case, but it's easier to just always follow the rule instead of analyzing the code each time to see what you can get away with for no additional benefit.

Reply:
For some reason assertThrows doesn't seem to exist in JUnit4 only in JUnit5

Reply:
ACK.